prepareRelease.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+declare nextVersion=$1
+
+sed -i.bak -e "s/.setVersion('.*')/.setVersion('$nextVersion')/g" src/main.ts
```

Comment:
This should update the version in the released tags. (Remembered that @polymath-eric wanted this wrt OTC, maybe it's not needed but I would like to keep this as well so that anyone spinning a new instance would get to know easily)

Reply:
(nit) I am always a bit worried when making raw text changes like this, especially when its changing code

Instead of: `.setVersion('1.0');` what about `.setVersion(REST_API_VERSION)`

and then target that variable to replace where its the top of the file (or even its own dedicated file)

That said, what you have is probably OK, but we don't have any release stopping test coverage on main.ts in either.